### ISSUE TITLE
Login: English text on the French page

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#41]
URL: https://hcdev.openplus.ca/fr/user/login affects Reset password as well. 
User Agent: Chrome
Language: French

Issue Description: 
"About this site" 
"Top of page" at the bottom is in English on the French page
"Symbol of the Government of Canada" alternative text is in English
"Government of Canada" logo at the top - alternative text is in English
![image](https://user-images.githubusercontent.com/65728700/128411104-8709d74f-6ee1-4385-aa5d-1e790f8b1345.png)

![image](https://user-images.githubusercontent.com/65728700/128414587-a5e069d5-258e-4589-a517-c40d0aee5c69.png)

Steps to reproduce:
View the page in Chrome. 
Run WAVE browser extension 
Remove styles 

Expected results:
These items should be translated. as per template.

**Comment by qanitah** [comment]
This seems to be template issue as this text appears to be in English on all French pages.

**Comment by ndolter** [comment]
Validated that the translations have been made.
![image](https://user-images.githubusercontent.com/68117281/135894767-c6fa30f8-753e-4524-b3bb-5a4a65d5d8e0.png)
